openapi/data_repository_service.swagger.yaml

```diff
@@ -21,7 +21,7 @@ security:
   - {}
   - authToken: []
 paths:
-  '/objects/{object_id}':
+  '/objects/{scheme}{identifier}/{object_id}':
```

Comment:
I think this line means that this PR is a 100% breaking change, since all DRS requests going forward MUST include an extra path component before the `object_id` -- is that right?